src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -211,6 +214,12 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
     startBlock: number,
     endBlock: number
   ): Promise<TransactionReceipt> {
+    // Do not submit batch if gas price above threshold
+    const gasPriceInGwei = gasInGwei(await this.signer.getGasPrice())
+    if (gasPriceInGwei > this.gasThresholdInGwei) {
+      return
```

Comment:
Could we do a `log.info` here with something along the lines of:
`Current gas price ${gasPriceInGwei} is higher than gas price threshold ${gasThresholdInGwei}`